### DISCUSSION TITLE
v24.2.0

### DISCUSSION THREAD

**Discussion by audgirka** [ansible/ansible-lint#4013] [Announcements]
## Minor Changes

- Drop support for Python 3.9 (#4009) @audgirka

## Bugfixes

- Fix scenario when role has no `dependencies` section in `meta/main.yml` (#3993) @audgirka

<hr /><em>This discussion was created from the release <a href='https://github.com/ansible/ansible-lint/releases/tag/v24.2.0'>v24.2.0</a>.</em>

**Comment by karl-johan-grahn** [comment]
Why a bump from 6.22.2 to 24.2.0? 🤔

**Reply by audgirka** [reply]
Please read: https://forum.ansible.com/t/devtools-projects-transitioning-to-calver-releases/3444
